cluster/addons/addon-manager/kube-addons.sh

```diff
@@ -155,7 +155,7 @@ function is_leader() {
   fi
   KUBE_CONTROLLER_MANAGER_LEADER=`${KUBECTL} -n kube-system get ep kube-controller-manager \
     -o go-template=$'{{index .metadata.annotations "control-plane.alpha.kubernetes.io/leader"}}' \
-    | sed 's/^.*"holderIdentity":"\([^"]*\)".*/\1/'`
+    | sed 's/^.*"holderIdentity":"\([^"]*\)".*/\1/' | awk -F'_' '{print $1}'`
```

Comment:
@mikedanese @liggitt do updates to this script still have to be merged first and then the pull that needs the update will be green?

Reply:
no idea